src/CompanyService.Data.Provider.MsSql.Ef/Migrations/20210705102500_AddSmtpToCompanyAndHistoryTable.cs

```diff
@@ -0,0 +1,89 @@
+﻿using LT.DigitalOffice.CompanyService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.CompanyService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(CompanyServiceDbContext))]
+    [Migration("20210705102500_AddSmtpToCompanyAndHistoryTable")]
+    public class AddSmtpToCompanyAndHistoryTable : Migration
+    {
+        private void UpdateCompanyTable(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Host",
+                table: DbCompany.TableName,
+                nullable: false);
```

Comment:
I think you need to add default value

Reply:
I can only make it nullabe, i don't have default value

Reply:
made it all nullable true